HACKING.md

```diff
@@ -18,23 +18,27 @@ dnf install -y oci-kvm-hook # if on traditional
 ```
 
 We will use `oc cluster up` to set up a local cluster for testing. To do
-this, simply obtain the OpenShift binary from
-[the latest release](https://github.com/openshift/origin/releases/latest)
-(though note that I'm currently still using the binaries from v3.9, so
-pick that release if you hit issues).
+this, obtain the OpenShift binary from
+[the v3.9.0 release](https://github.com/openshift/origin/releases/tag/v3.9.0).
+`openshift-origin-client-tools` is enough for `oc cluster up`.
+
+If you have not done so already, follow the instructions in the
+[Prerequisites](https://github.com/openshift/origin/blob/v4.0.0-alpha.0/docs/cluster_up_down.md#prerequisites)
+and [Getting Started](https://github.com/openshift/origin/blob/v4.0.0-alpha.0/docs/cluster_up_down.md#getting-started)
+sections to set up your environment and start the Docker daemon.
 
 And now, bring up a v3.6.1 cluster (we want to match the CentOS CI
 version):
 
 ```
-oc cluster up --version v3.6.1
+$ oc cluster up --version v3.6.1
```

Comment:
This is minor, though I tend to not include the `$` in instructions to make it easier to copy paste (on most setups, a triple-click will select the full paragraph). I admit this is slight bikeshed though. :)

Reply:
i sit firmly on the fence. A lot of old manuals use `$` or `#` to indicate normal user vs root user. If we're not trying to have that effect here I'm fine with leaving them out.

Reply:
Ahh yeah, that is easier. I'll remove the `$` :+1:

Reply:
Sorry @dustymabe  your comment hadn't appeared on my end yet when I first saw. Agree, they are handy for showing the root/non-root difference. Agreed with for this situation they can be left out.